test/command/upload.spec.ts

```diff
@@ -31,6 +31,18 @@ describeCommand('Test Upload command', ({ consoleMessages, hasMessageContaining
     expect(command.hash?.length).toBe(64)
   })
 
+  it('should upload file and encrypt', async () => {
```

Comment:
the output files should be checked at least whether they differ.

Reply:
Hmm, if you do not pass the encryption key, then the content is 404.

If you do pass it, then you get back the original file.

I am not sure if this is worth testing.

Reply:
https://github.com/ethersphere/bee/issues/2215